client/package.json

```diff
@@ -102,10 +102,10 @@
     "moment": "2.10.6",
     "moment-timezone": "^0.5.5",
     "ng-file-upload": "^12.2.11",
-    "ng-sir-trevor": "git+https://github.com/superdesk/ng-sir-trevor",
+    "ng-sir-trevor": "superdesk/ng-sir-trevor",
     "rangy": "^1.3.0",
     "raven-js": "^3.9.1",
-    "sir-trevor": "git+https://github.com/m0g/sir-trevor-js#liveblog",
-    "superdesk-core": "git+https://github.com/m0g/superdesk-client-core#moment-fix"
+    "sir-trevor": "m0g/sir-trevor-js#liveblog",
+    "superdesk-core": "nistormihai/superdesk-client-core"
```

Comment:
Is it not OK to use personal repos for this. We all have access to SF's liveblog repo so please use that!